src/main/java/org/prebid/server/bidder/ix/IxAdapter.java

```diff
@@ -81,9 +81,13 @@ private static void validatePreBidRequest(PreBidRequest preBidRequest) {
 
         for (AdUnitBid adUnitBid : adUnitBids) {
             final IxParams ixParams = parseAndValidateParams(adUnitBid);
+            final List<Integer> ixParamsSizes = ixParams.getSizes();
             boolean isFirstSize = true;
-            for (Format size : adUnitBid.getSizes()) {
-                final BidRequest bidRequest = createBidRequest(adUnitBid, ixParams, size, preBidRequestContext);
+            for (Format format : adUnitBid.getSizes()) {
```

Comment:
Can `adUnitBid.getSizes()` be NULL ?

Reply:
Looks like it can't - I've checked up with other adapters and there are no NULL checks where `adUnitBid.getSizes()` is called. So I guess we'll let it stay as it is.